src/js/angular/ttyg/templates/chat-item-detail.html

```diff
@@ -95,3 +103,22 @@
     <div ng-if="asking" onto-loader-new size="40" guide-selector="question-loader"></div>
 </div>
 
+<script type="text/ng-template" id="token-usage-info.html">
+    <div class="title">{{'ttyg.chat_panel.dialog.token-usage-info.title' | translate}}</div>
```

Comment:
`token-usage-info` use underscores for consistency

Reply:
done